app/screens/Reusable/Timers/TimerTitleButton/TimerTitleButton.tsx

```diff
@@ -35,20 +36,19 @@ export const TitleButton: React.FC<ITitleButtonProps> = ({
 
   return (
     <SafeAreaView style={generalStyles.flex}>
-      <MainHeader leftIcon={IMAGES.WHITE_BACK_ARROW} />
+      <MainHeader
+        leftIcon={IMAGES.WHITE_BACK_ARROW}
+        onLeftIconPress={navigation.goBack}
+      />
       <BottomButtonView
         buttonTitle={t(correctButtonTitle)}
         onSubmit={onSubmitPress}
         isDisabledButton={isTimerStart}
         style={styles.container}>
-        <Timer
-          duration={duration}
-          isStart={isTimerStart}
-          onAnimationComplete={onSubmit}
-        />
-        {!!titleKey && (
+        <Timer duration={duration} isStart={isTimerStart} />
+        {title && (
           <ExtendedText preset="heading" style={styles.title}>
-            {t(titleKey)}
+            {t(title)}
```

Comment:
`titleKey` is more preferable here

Reply:
Anna said to use these names for props:
- title
- description
- tellMoreTitle
- tellMoreDescription
- backgroundImage
- tellMoreBackgroundImage
- description?